proto/dataplane/dataplane.proto

```diff
@@ -0,0 +1,132 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+package lemming.dataplane;
+
+option go_package = "github.com/openconfig/lemming/proto/dataplane";
+
+message UpdatePortRequest {
+  string name = 1;
+  string hwaddr = 2;
+  repeated IPNetwork ipv4 = 3;
+  repeated IPNetwork ipv6 = 4;
+}
+
+message UpdatePortResponse {}
+
+enum PortState {
+  PORTSTATE_INVALID = 0;
+  PORTSTATE_DOWN = 1;
+  PORTSTATE_UP = 2;
+}
+
+message SetPortStateRequest {
+  string name = 1;
+  PortState admin_state = 2;
+}
+
+message SetPortStateResponse {}
+
+message SubscribePortStateRequest {}
+
+message IPNetwork {
+  string ip = 1;
+  uint64 prefix_len = 3;
+}
```

Comment:
(no action required) It looks like there is some overlap between our respective protos that we can consolidate later. I'll mark a TODO in my PR for this.

Reply:
ack, a proto package with common types seems like a good idea